pkgs/applications/misc/hello/default.nix

```diff
@@ -1,17 +1,18 @@
-{ lib
+{ callPackage
+, lib
 , stdenv
 , fetchurl
 , nixos
 , testers
 , hello
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
https://github.com/NixOS/nixpkgs/pull/119942/files#diff-c032817f6a62e23767f4c7706e35eef36e8353aa5ba5d6f9bce8c6a8cc0db01dR53

>  If only a one-argument function is written, the argument has the meaning of `previousAttrs`.

so this should be 
`finalAttrs: previousAttrs:`
right?

Reply:
ah nevermind thats for `overrideAttrs` :sweat_smile:

Reply:
That is what `overrideAttrs` takes, `stdenv.mkDerivation` is the ground floor so there are no `previousAttrs`.